autosklearn/smbo.py

```diff
@@ -366,7 +366,7 @@ def run_smbo(self):
         # first create a scenario
         seed = self.seed
         self.config_space.seed(seed)
-        num_params = len(self.config_space.get_hyperparameters())
+        self.config_space.get_hyperparameters()
```

Comment:
I removed the variable assignment, but shouldn't we get rid of the line altogether?

Reply:
If it's not used you can remove it.

Reply:
removed